### ISSUE TITLE
ORT script failure due to TASK-related variable changes

### ISSUE THREAD

**Issue by zach1221** [ufs-community/ufs-weather-model#1586]
Description

On tier one HPCs Hera and Orion, the operational requirement script fails with the below error.
"/ufs-weather-model/tests/opnReqTests/std.sh: line 34: TASKS: unbound variable Died with error code 1"

Expected behavior is that ORT should run successfully and pass ORT tests.

To Reproduce:

1. Log into Hera or Orion 
2. cd into working directory 
3. run git clone of ufs-community/ufs-weather-model develop branch
4. cd into ufs-weather-model/tests
5. run ./opnReqTest -n control_p8 -c std

Additional context

This error was also produced by running cases regional_control and cpld_control_nowave_noaero_p8. As well as with all ORT tests std,thr, mpi, dcp, rst, bit, dbg

The issue was possibly created by PR #1523 as there were changes to TASK-related variables from 1523. ORT script update is needed to reflect the change introduced from #1523.

![image](https://user-images.githubusercontent.com/99902696/214976927-f7e828d4-7cca-44dd-a88e-af761c2a9eea.png)

**Comment by jkbk2004** [comment]
@zach1221 can you provide error messages?

**Comment by zach1221** [comment]
@jkbk2004 I did but my content isnt showing above. You see it if you select edit.

**Comment by zach1221** [comment]
## Description



### To Reproduce:



## Additional context
<!--

This error was also produced by running cases regional_control and cpld_control_nowave_noaero_p8. As well as with all ORT tests std,thr, mpi, dcp, rst, bit, dbg

The issue was possibly created by PR #1523 as there were changes to TASK-related variables from 1523. ORT script update is needed to reflect the change introduced from #1523.

![image](https://user-images.githubusercontent.com/99902696/214976642-691bfce8-0c19-4a40-a799-537002396e8b.png)

**Comment by zach1221** [comment]
Hi, @DusanJovic-NOAA 

We discovered an issue with running ORT scripts on hera/orion, mentioned above. We believe the error is caused by changes made to TASKS variables in PR #1523 . To account for these updates, I tested removing TASKS variables from opnReqTest, wrt_env.sh, dbg.sh, dcp.sh, mpi.sh, rst.sh, std.sh, thr.sh and the below tests pass successfully.

regional_control std,bit,dcp,thr
cpld_control_nowave_noaero_p8 std,bit,dbg,dcp,rst,thr
control_p8 std,bit,dbg,dcp,fhz,mpi,rst,thr

However, we'd like to get your input on this as well, before committing any changes. Are there any other updates we should make to other files that may impact ORT? If useful, I can post the updates I made while testing.
